src/components/HighlightChat/HighlightChat.tsx

```diff
@@ -39,14 +40,20 @@ function useHandleClipboardPaste() {
                 value: URL.createObjectURL(file),
                 file: file,
               });
+              trackEvent('hl_chat_image_pasted', { fileType: file.type });
             } else if (file.type === "application/pdf") {
               console.log("Pasted PDF");
               addAttachment({
                 type: "pdf",
                 value: file,
               });
+              trackEvent('hl_chat_pdf_pasted', {});
+            } else {
+              trackEvent('hl_chat_unsupported_file_pasted', { fileType: file.type });
             }
           }
+        } else if (item.kind === "string") {
+          trackEvent('hl_chat_text_pasted', {});
```

Comment:
These can be consolidated to a `Chat Paste Item` event with a `type` property

Reply:
Done